fe/fe-core/src/main/java/com/starrocks/metric/MetricRepo.java

```diff
@@ -118,15 +119,21 @@ public final class MetricRepo {
     public static GaugeMetricImpl<Long> GAUGE_MAX_TABLET_COMPACTION_SCORE;
     public static GaugeMetricImpl<Long> GAUGE_STACKED_JOURNAL_NUM;
 
-    private static ScheduledThreadPoolExecutor metricTimer =
+    public static List<GaugeMetricImpl<Long>> GAUGE_ROUTINE_LOAD_LAGS;
+    public static ReentrantReadWriteLock GAUGE_ROUTINE_LOAD_LAGS_LOCK;
```

Comment:
The update of reference is atomic, so this lock seems useless.

Reply:
Fixed